protos/encodings.proto

```diff
@@ -218,6 +218,11 @@ message PackedStruct {
   Buffer buffer = 2;
 }
 
+message FixedSizeBinary {
+  ArrayEncoding bytes = 1;
+  uint32 byte_width = 2;
+}
```

Comment:
Why is a new message needed?  How is this different from `Flat`?

Reply:
Flat uses a buffer, but we need an ArrayEncoding? Since bytes is actually a BasicEncoder(ValueEncoder())
Otherwise it's hard to reconstruct the BasicPageScheduler directly? We would need to reconstruct the ArrayEncoding from the buffer...